server/httpin/restapi/call.go

```diff
@@ -44,15 +44,6 @@ func (a *restapi) Call(w http.ResponseWriter, req *http.Request, in proxy.Incomi
 
 	res := a.proxy.Call(in, *creq)
 
-	a.conf.Logger().Debugw(
-		"Received call response",
-		"app_id", creq.Context.AppID,
-		"acting_user_id", in.ActingUserID,
-		"error", res.ErrorText,
-		"type", res.Type,
-		"path", creq.Path,
-	)
-
```

Comment:
Why was this removed?

Reply:
I found this particular log to be useless, spammy, and verbose on Text responses.

Reply:
2/ I kind of liked it as it give a clear notice that and what response the apps plugin got